hooks.lua

```diff
@@ -160,7 +160,9 @@ end
 
 function irc.hooks.kick(channel, target, prefix, reason)
 	if target == irc.conn.nick then
-		minetest.chat_send_all("IRC: kicked from "..channel.." by "..prefix.nick..".")
+		local message = "IRC: kicked from "..channel.." by "..prefix.nick.."."
+		minetest.chat_send_all(message)
+		irc:logChat(message)
```

Comment:
This will result in something like `IRC CHAT: IRC: kicked ...` being printed.  The double `IRC` is redundant and it isn't really part of the chat, so it should probably just be directly logged through `minetest.log`.